### ISSUE TITLE
Problem using conda build on windows

### ISSUE THREAD

**Issue by kikocorreoso** [takluyver/flonda#2]
Hi, 

I'm using windows to upload a package to anaconda and I get the following error:

```python
Packages are now in dist\*\tutormagic-0.3.0-*.tar.bz2
Uploading 5 files to anaconda.org...
Traceback (most recent call last):
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "d:\users\my_user\AppData\Local\Continuum\Miniconda3\Scripts\flonda.exe\__main__.py", line 9, in <module>
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\site-packages\flonda\cli.py", line 47, in main
    publish_packages(files)
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\site-packages\flonda\cli.py", line 81, in publish_packages
    subprocess.run(['anaconda', 'upload'] + package_files, check=True)
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\subprocess.py", line 403, in run
    with Popen(*popenargs, **kwargs) as process:
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\subprocess.py", line 964, in _execute_child
    args = list2cmdline(args)
  File "d:\users\my_user\appdata\local\continuum\miniconda3\lib\subprocess.py", line 461, in list2cmdline
    needquote = (" " in arg) or ("\t" in arg) or not arg
TypeError: argument of type 'WindowsPath' is not iterable
```

On Linux doing the same I can upload to files to anaconda.

**Comment by takluyver** [comment]
Does it work if you change this line:
https://github.com/takluyver/flonda/blob/de3dccece4e915c2b59266f1fb9869c4aa1bdd36/flonda/cli.py#L81

To something like:

```python
subprocess.run(['anaconda', 'upload'] + [str(f) for f in package_files], check=True)
```

**Comment by kikocorreoso** [comment]
It works like a charm modifying that line 😀 

Do you want a PR 😛 ?

Thanks.

**Comment by takluyver** [comment]
Yes please :-)

**Comment by takluyver** [comment]
Fixed by #4
